Capstone/Like View/LikeViewController.m

```diff
@@ -0,0 +1,78 @@
+//
+//  LikeViewController.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/12/22.
+//
+
+#import "LikeViewController.h"
+#import <Parse/Parse.h>
+#import "APIManager.h"
+#import "College.h"
+#import "LikeCell.h"
+#import "DetailsViewController.h"
+
+@interface LikeViewController () <UITableViewDelegate, UITableViewDataSource>
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@end
+
+@implementation LikeViewController {
+    NSArray *likedCollegeNames;
+    NSArray<College *> *colleges;
+    NSMutableArray *likedCollegesToDisplay;
+}
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+    self->likedCollegeNames = [[NSArray alloc] init];
```

Comment:
```suggestion
```

Line isn't necessary if you're replacing this empty array with `current[@"likes"]` below